Order Fulfillment with MIP/README.md

```diff
@@ -0,0 +1,46 @@
+# Nextmv MIP knapsack template
+
+This template demonstrates how to solve a Mixed Integer Programming problem
+using the open-source solver [HiGHS](https://github.com/ERGO-Code/HiGHS). To run
+this beta template, please contact support@nextmv.io for access to the Nextmv
+HiGHS interface.
+
+To solve a Mixed Integer Problem (MIP) is to optimize a linear objective
+function of many variables, subject to linear constraints. We demonstrate this
+by solving the knapsack problem.
+
+Knapsack is a classic combinatorial optimization problem. Given a collection of
+items with a value and weight, our objective is to maximize the total value
+without exceeding the weight capacity of the knapsack.
+
+The input defines a number of items which have an id to identify the item, a
+weight and a value. There is also a volume, which is currently unused in the
+template but can be used to extend the template to a multi-dimensional knapsack.
+Additionally there are two capacities, one for weight and one for volume (again,
+the latter one is not used in this template).
```

Comment:
I think we should remove any mentioning of Knapsack in the readme or remove it entirely if that is easier

Reply:
I changed the readme and removed knapsack from it. The only place left, where knapsack is referenced, is the code comment you saw in the main.go file.